apps/felles/src/main/kotlin/no/nav/helsearbeidsgiver/felles/domene/AktiveArbeidsgivere.kt

```diff
@@ -1,16 +1,17 @@
 package no.nav.helsearbeidsgiver.felles.domene
 
 import kotlinx.serialization.Serializable
+import no.nav.helsearbeidsgiver.utils.wrapper.Orgnr
 
 @Serializable
 data class AktiveArbeidsgivere(
-    val fulltNavn: String? = null,
-    val avsenderNavn: String,
-    val underenheter: List<Arbeidsgiver>,
+    val sykmeldtNavn: String?,
```

Comment:
Ingen big deal for meg, men fulltNavn er mer "nøytralt" - klienten kan jo i teorien benyttes til å slå opp arbeidsgiver for hvem som helst, nå blir en arbeidstaker per def sykmeldt. Kanskje er arbeidstaker alltid sykmeldt i denne konteksten, men tja.